jobs/JGDAS_GLOBAL_MARINE_ANALYSIS_LETKF

```diff
@@ -12,9 +12,14 @@ GDATE=$(date --utc +%Y%m%d%H -d "${PDY} ${cyc} - ${assim_freq} hours")
 gPDY=${GDATE:0:8}
 gcyc=${GDATE:8:2}
 
+export GDUMP=${GDUMP:-"gdas"}
```

Comment:
```suggestion
export GDUMP="gdas"
```
This is the first place `GDUMP` is defined and so there is no incoming value for it.

Reply:
done